count-buttons.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+// Using useState as a hook.
+
+export default function App() {
+
+    const [count, setCount] = useState(0);
+    // Making the the array variable with useState.
```

Comment:
Nice work. 

BONUS work:
Mind if I challenge you to print the string "count changed" to the console upon the change of the count state variable?